app/javascript/styles/mastodon/rtl.scss

```diff
@@ -174,6 +163,10 @@ body.rtl {
     margin-right: 0;
   }
 
+  .detailed-status__display-name .display-name {
+    text-align: right;
```

Comment:
I don't think this is needed here. If rtl styles are loaded, then the text here will be dir=rtl (by inheritance) and align=left (I assume by CSS convention). No?